hypertrace-trace-enricher/trace-reader/src/main/java/org/hypertrace/trace/reader/attributes/DefaultValueResolver.java

```diff
@@ -30,7 +30,7 @@
 
   @Override
   public Single<LiteralValue> resolve(
-      ValueSource valueSource, AttributeMetadata attributeMetadata) {
+      ValueSource<?, ?> valueSource, AttributeMetadata attributeMetadata) {
```

Comment:
all the wildcards certainly indicate me there's no value in having a `ValueSource` be parameterized